### PR TITLE
readme suggestions

### PR DESCRIPTION
**Pull request by rockem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,37 +11,28 @@ artifacts available @ [*bintray repositories*](https://bintray.com/ticktok-io/ma
 ```
 dependencies {
   ...
-    compile 'io.ticktok:ticktok-java-client:<version>'
+    compile 'io.ticktok:ticktok-java-client:0.3.0'
```
Comment (amaziagur) [inline README.md:14]: should be generic, this is why i added a link to the bintray

File: README.md [hunk 2]
```diff
@@ -16,37 +11,28 @@ artifacts available @ [*bintray repositories*](https://bintray.com/ticktok-io/ma
...
 
 ```
-
 ### Maven
 ```
 <dependency>
   <groupId>io.ticktok</groupId>
   <artifactId>ticktok-java-client</artifactId>
-  <version>_version_</version>
+  <version>0.3.0</version>
```
Comment (amaziagur) [inline README.md:23]: look above

**Review by amaziagur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rockem) [reply README.md:14]: it's pretty standard to specify latest version:
Few examples:
https://github.com/rabbitmq/rabbitmq-java-client
https://github.com/perwendel/spark
https://github.com/xetorthio/jedis

Comment (amaziagur) [reply README.md:14]: yes, but still i dont want to maintain.
really easy to follow the link...

Comment (rockem) [reply README.md:14]: Can't argue with that

**Pull request closed without merging**